UPGRADE.md

```diff
@@ -5,6 +5,16 @@
 When upgrading also have a look at the changes in the
 [sulu skeleton](https://github.com/sulu/skeleton/compare/2.0.2...2.0.3).
 
+### New methods for Form/ListViewBuilder
```

Comment:
Would write "New methods in ViewBuilderInterfaces", because otherwise it is not really a breaking change.